File: Backend/ProjektZespołówka/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjektZespołówka.Models
{
    public class Project
    {
        public int Id { get; set; }
        public string Title { get; set; } = null!;
        public string Description { get; set; } = null!;
        public string ContactPhone { get; set; } = null!;
        public string ContactEmail { get; set; } = null!;
        public Helpers.ProjectStatus Status { get; set; } = Helpers.ProjectStatus.Planned;
        public User Manager { get; set; } = null!;
        public User Investor { get; set; } = null!;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public Localization? Location { get; set; }
    }
}

File: mobile/composeApp/src/commonMain/kotlin/com/rackncode/cloudwire/domain/model/Project.kt
package com.rackncode.cloudwire.domain.model

import com.rackncode.cloudwire.domain.helpers.ProjectStatus

data class Project(
    val id: String,
    val title: String,
    val description: String,
    val contactPhone: String,
    val contactEmail: String,
    val status: ProjectStatus = ProjectStatus.PLANNED,
    val manager: User,
    val investor: User,
    val localization: Localization?
)
